PublicOnboardingDemo/clients/LandingPage/controllers/client/onStart.js

```diff
@@ -0,0 +1,32 @@
+    
+    var isPublic = client.isPublic();
+    
+    var intro = client.getWidget("WelcomeLine");
+    intro.text = isPublic ? "Welcome to VANTIQ! You are currently in Public Mode." : "Welcome to VANTIQ! You are currently in Private Mode.";
+    
+    var uuid = client.getUsername();
```

Comment:
Please change this variable name to username. We want all of the variables to match the concept/term in our system to be as clear to viewers as possible.

In general, that means switching uuid -> username and username -> preferred username (or user record in the case below)